File: attribouter/src/main/java/me/jfenn/attribouter/wedges/link/WebsiteLinkWedge.java
package me.jfenn.attribouter.wedges.link;

import androidx.annotation.NonNull;

public class WebsiteLinkWedge extends LinkWedge {

    public WebsiteLinkWedge(@NonNull String url, int priority) {
        super("website", "@string/title_attribouter_website", url, "@drawable/ic_attribouter_link", false, priority);
    }

}

File: attribouter/src/main/java/me/jfenn/attribouter/wedges/link/WebsiteLinkWedge.kt
package me.jfenn.attribouter.wedges.link

class WebsiteLinkWedge(
        url: String, priority: Int
) : LinkWedge(
        id = "website",
        name = "@string/title_attribouter_website",
        url = url,
        icon = "@drawable/ic_attribouter_link",
        priority = priority
)
